src/project-tests/scatter-plot-tests.js

```diff
@@ -315,6 +319,16 @@ export default function createScatterPlotTests() {
           document.getElementById('legend'),
           'There should be an element with id="legend" '
         );
+        var legendText;
+        if (document.querySelector('#legend text') !== null) {
```

Comment:
What's happening here? It's not clear to me why this `if` statement is needed? Is it cross browser compatibility, or something else? Point being, could you please put a comment about the statement to explain what's happening? :-) Thanks!